packages/api/src/auth/verifiers/index.ts

```diff
@@ -0,0 +1,141 @@
+import type { JwtVerifier } from './jwtVerifier'
+import { jwtVerifier } from './jwtVerifier'
+import type { SecretKeyVerifier } from './secretKeyVerifier'
+import { secretKeyVerifier } from './secretKeyVerifier'
+import type { Sha256Verifier } from './sha256Verifier'
+import { sha256Verifier } from './sha256Verifier'
+import { skipVerifier } from './skipVerifier'
+import type { SkipVerifier } from './skipVerifier'
+import type { TimestampSchemeVerifier } from './timestampSchemeVerifier'
+import { timestampSchemeVerifier } from './timestampSchemeVerifier'
+
+const typesToVerifiers = {
```

Comment:
Maybe we can move these common types to another file to prevent the circular dependency, as well as making it easier to read.